spec/request/api/v1/target/targets_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'Target /api/v1/targets', type: :request do
+  let(:user) { create(:user, confirmed_at: Time.zone.today) }
+  let!(:user2) { create(:user, confirmed_at: Time.zone.today) }
+  let(:topic) { create(:topic) }
+  let!(:target) do
+    create :target, user: user2, topic_id: topic.id,
+                    latitude: -32.528879, longitude: -55.769835, radius: 400
+  end
+
+  before(:each) do
+    login
+  end
+
+  subject do
+    post '/api/v1/targets', params: { target: { title: 'target title',
+                                                topic_id: topic.id,
+                                                latitude: -32.528879,
+                                                longitude: -55.769835,
+                                                radius: 400.0 } },
+                            headers: auth_headers
+  end
+
+  it 'should return a successful response' do
+    subject
+    expect(response).to be_successful
+  end
+
+  it 'should return status created' do
+    subject
+    expect(response).to have_http_status(:created)
+  end
+
+  it 'should creates the target' do
+    expect {
+      subject
+    }.to change(Target, :count).by(1)
+  end
+
+  it 'should returns the created target' do
+    subject
+    expect(json['target']['title']).to eq('target title')
```

Comment:
You can use `let` to define variables in rspec.

See https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let